bindings_generator/src/api.rs

```diff
@@ -84,6 +85,9 @@ impl GodotClass {
     }
 }
 
+pub type ConstantName = String;
```

Comment:
I don't know why these type aliases are needed. Perhaps you wanted newtypes for type safety...?

Reply:
I just put type aliases so the HashMap is a bit easier to read meaning into. I did not really aim for strongly typed wrappers. I could make them newtypes if you think it's worth, I just preferred this over a comment :D

Reply:
I see! I think the String should be pretty self-explanatory, but I agree that a bare `i64` can be a bit hard to make sense of.

I don't think they have to be newtypes. Maybe `ConstantValue` can be an enum if you want, in case constants of other types get added in the future, but at this point it probably only adds complexity. I think it's fine either way.